app/admin/users.rb

```diff
@@ -13,6 +13,10 @@
     link_to 'Change Password', change_password_admin_user_path(user)
   end
 
+  action_item :invite, only: [:edit, :show] do
+    link_to 'Invite', invite_admin_user_path(user) unless user.joined?
```

Comment:
I think this ultimately can be `if user.lead?`. Once they actually set a password, then we can change their account type to `basic`.

Reply:
Couldn't we have someone in the system that's not really a lead but needs an invite? ie, another `admin` on the team that we're adding so we can invite to the system?